tools/benchmark/package.json

```diff
@@ -41,7 +41,8 @@
 		"easy-table": "^1.2.0",
 		"mocha": "^10.0.0",
 		"mocha-multi-reporters": "^1.5.1",
-		"moment": "^2.21.0"
+		"moment": "^2.21.0",
+		"picocolors": "^1.1.1"
```

Comment:
If we end up removing the stdout/stderr writes, I think this becomes unnecessary.